### PR TITLE
Benchmarks: Update - Update benchmarks in configuration file

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Update benchmarks in configuration files for single node validation of superbench v0.3.

**Major Revision**
- fix bugs of parameters in nccl-bw for single node validation in configs
- update new benchmarks in amd_mi100_hpe.yaml, amd_mi100_z53.yaml, azure_ndv4.yaml
- fix bug of wrong gpu visible prefix

### CODE REVIEW

File: superbench/config/azure_ndv4.yaml [hunk 1]
```diff
@@ -35,6 +35,32 @@ superbench:
       <<: *default_local_mode
     gemm-flops:
       <<: *default_local_mode
+    nccl-bw:
+      enable: true
+      modes:
+        - name: local
```
Comment (abuccts) [inline superbench/config/azure_ndv4.yaml:41]: why rccl-bw uses mpi mode but nccl-bw uses local?

**Review by abuccts — COMMENTED** [review]
could you also update mi100 z53 config?

**Review by cp5555 — APPROVED** [review]

**Review by abuccts — APPROVED** [review]

File: superbench/config/amd_mi100_hpe.yaml [hunk 2]
```diff
@@ -40,16 +40,14 @@ superbench:
     rccl-bw:
```
Comment (cp5555) [inline superbench/config/amd_mi100_hpe.yaml:40]: Would you please revise the description?

Comment (yukirora) [reply superbench/config/amd_mi100_hpe.yaml:40]: updated

**Pull request merged**